atoMEC/__init__.py

```diff
@@ -168,6 +156,8 @@ def units_density(self, units_density):
 
     @property
     def radius(self):
+        r"""float: radius of the Wigner-Seitz sphere, defined as :math:`0.5*a_i`,
```

Comment:
Either omit the `*` symbol or rewrite equation to
```suggestion
        r"""float: radius of the Wigner-Seitz sphere, defined as :math:`a_i /2`,
```